pyteal.ini

```diff
@@ -0,0 +1,5 @@
+[pyteal]
```

Comment:
Discussion point: I'm introducing this file so that source mapper can be turned off completely.